tests/nn/data_parallel/test_fsdp.py

```diff
@@ -268,6 +276,7 @@ def rename_test(testcase_func, param_num, param):
     return "%s_%s" % (testcase_func.__name__, parameterized.to_safe_name(str(param.args)),)
 
 
+@pytest.mark.skipif(torch_version() < (1, 9, 0), reason="pytorch version >= 1.9.0 required")
```

Comment:
should we add a class level pytest skip annotation? maybe I missed that another test class is being run that does not call the `_train_for_several_steps ` function.

Reply:
I don't follow this comment. can you please elaborate?